tests/test_manylinux.py

```diff
@@ -170,7 +170,10 @@ def test_build_wheel_with_binary_executable(docker_container):
     manylinux_id, python_id, io_folder = docker_container
     docker_exec(manylinux_id, 'yum install -y gsl-devel')
 
-    docker_exec(manylinux_id, ['bash', '-c', 'cd /auditwheel_src/tests/testpackage && python setup.py bdist_wheel -d /io'])
+    docker_exec(manylinux_id, ['bash', '-c',
+                               'cd /auditwheel_src/tests/testpackage',
+                               'python setup.py bdist_wheel -d /io']
+                )
```

Comment:
This change is not semantically equivalent and should be reverted. It's not showing up as test breakage because something else appears to be causing the tests to fail.